package.json

```diff
@@ -64,7 +64,6 @@
     "@ngtools/webpack": "7.3.9",
     "@pact-foundation/karma-pact": "2.2.0",
     "@pact-foundation/pact-node": "9.0.0",
-    "@skyux/auth-client-factory": "1.1.0",
```

Comment:
Is something else providing it?

Reply:
Yes, see the pull request's description. ;-)

Reply:
I did not.  :-)